src/api/config/routes/webui_routes.rb

```diff
@@ -66,7 +66,8 @@
         get 'package/revisions/:project/:package' => :revisions, constraints: cons, as: 'package_view_revisions'
         post 'package/submit_request/:project/:package' => :submit_request, constraints: cons
         get 'package/rdiff/:project/:package' => :rdiff, constraints: cons, as: 'package_rdiff'
-        post 'package/save_new/:project' => :save_new, constraints: cons, as: 'save_new_package'
+        post 'package/create/:project' => :create, constraints: cons, as: 'packages'
```

Comment:
This would be better if we used resources, but it involves a lot more changes due to the huge amount of routes defined here.